packages/flutter/README.md

```diff
@@ -7,110 +7,3 @@ focused on low-latency input and high frame rates on Android and iOS.
-
-In a large app, different modules or packages might be bundled with their own localizations. That's why the Localizations widget manages a table of objects, one per LocalizationsDelegate. To retrieve the object produced by one of the LocalizationsDelegate's `load` methods, you specify a BuildContext and the object's type.  For example the localized strings for the Material widgets are defined by the `MaterialLocalizations` class. Instances of this class are created by a LocalizationDelegate provided by the `MaterialApp` class. They can be retrieved with `Localizations.of`:
-
-```dart
-Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
-```
-
-This particular `Localizations.of()` expression is used frequently, so the MaterialLocalizations class provides a convenient shortand:
-
-```dart
-static MaterialLocalizations of(BuildContext context) {
-  return Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
-}
-
-/// References to the localized values defined by MaterialLocalizations
-/// are typically written like this:
-
-tooltip: MaterialLocalizations.of(context).backButtonTooltip,
-```
-
-The example that follows can be viewed as a template for using Flutter's localizations classes. It defines a single class, MyLocalizations, that contains the app's localized resources (just one for the example). It uses the initializeMessages() function generated by the Dart intl package to load resources and Intl.message() to look them up.
-
-```dart
-class MyLocalizations {
-  MyLocalizations(this.locale);
-
-  final Locale locale;
-
-  static Future<MyLocalizations> load(Locale locale) {
-    return initializeMessages(locale.toString())
-      .then((Null _) {
-        return new MyLocalizations(locale);
-      });
-  }
-
-  static MyLocalizations of(BuildContext context) {
-    return Localizations.of<MyLocalizations>(context, MyLocalizations);
-  }
-
-  String title() => Intl.message('<title>', name: 'title', locale: locale.toString());
-  // ... more Intl.message() methods like title()
-}
-```
-
-A class based on the `intl` package imports a generated message catalog that provides
-the `initializeMessages()` function and the per-locale backing store for `Intl.message()`.
-The message catalog is produced by an `intl` tool that analyzes the source code for
-classes that contain `Intl.message()` calls. In this case that would just be the
-`MyLocalizations` class.
-
-An instance of `MyLocalizations` is created by a simple LocalizationsDelegate:
-
-```dart
-class MyLocalizationsDelegate extends LocalizationsDelegate<MyStrings> {
-  @override
-  Future<MyStrings> load(Locale locale) => MyStrings.load(locale);
-
-  @override
-  bool shouldReload(MyLocalizationsDelegate old) => false;
-}
-```
-
-An instance of `MyLocalizationsDelegate` is provided to application's MaterialApp (or its WidgetsApp for non-material applications):
```

Comment:
MaterialApp and WidgetsApp should have backticks around them (and be links to the docs, but you already said you were going to do that for class names).

Reply:
I should have [bracketed them], thanks.